src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -205,9 +208,22 @@ export const ImportAccounts = ({ chainId, onClose }: ImportAccountsProps) => {
     enabled: queryEnabled,
   })
 
+  const supportsMultiAccount = useMemo(() => {
+    if (!wallet?.supportsBip44Accounts()) return false
+    if (!accounts) return false
```

Comment:
Will need to test if this still works after removing all accounts from MM with snaps and refreshing the app

Reply:
Unable to test because of https://discord.com/channels/554694662431178782/1242670834930290688/1246119118197756025

But not an issue either way as this is only used as a directive to disable the button, and not as a condition for the query to run 🎉